winget-install.ps1

```diff
@@ -156,24 +156,48 @@ function Add-ScopeMachine {
 }
 
 #Check if app is installed
-function Confirm-Install ($AppID) {
-    #Get "Winget List AppID"
-    $InstalledApp = & $winget list --Id $AppID -e --accept-source-agreements | Out-String
+function Confirm-Install ($AppID, $AppArgs) {

```

Comment:
In fact, the `Confirm-Install` function is not really necessary in a SCCM/Intune context as we have the detection script. Not sure it is really useful to keep.